ci: fix PR labeler

rickstaa opened the pull request:
**What does this pull request do? Explain your changes. (required)**
<!-- A clear and concise description of what this pull request does. -->

This pull request ensures that the PR labeler action is working as expected again.

**Specific updates (required)**
<!--- List out all significant updates your code introduces -->
- 
- 
- 

**How did you test each of these updates (required)**
<!-- A detailed description of how you tested your code changes. Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code, etc. -->


**Does this pull request close any open issues?**
<!-- Fixes # -->

**Checklist:**
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] Read the [contribution guide](./CONTRIBUTING.md)
- [x] `make` runs successfully
- [x] All tests in `./test.sh` pass
- [x] README and other documentation updated
- [ ] ~[Pending changelog](./CHANGELOG_PENDING.md) updated~

rickstaa committed:
ci: fix PR labeler

This commit ensures that the PR labeler action is working as expected
again.

j0sh commented:
does this fix work?

<img width="914" alt="image" src="https://github.com/user-attachments/assets/0bce5e6f-8b0a-4457-ac01-4b7e9f6f16a8">

rickstaa commented:
@j0sh, @leszko thanks for your review. I see I made one typo in the labels but appart from that it should work. Will fix today 👍🏻.

rickstaa committed:
fix: update labeler cfg to new format

This commit ensures that we use the labeler config format that was
introduced in v5.

rickstaa commented:
@leszko sorry for the noise I didn't want to create a workflow trigger so just opened new pull request to test the changes. You can see it working [here](https://github.com/livepeer/go-livepeer/actions/runs/11895076841/job/33143794386?pr=3258).

rickstaa committed:
Merge branch 'master' into fix_labeler
